pkg/table/table.go

```diff
@@ -138,12 +145,24 @@ func (m *Model) SetRows(rows []Row) tea.Cmd {
 }
 
 func (m *Model) updateFilteredRows() {
+	rows := m.rows
+	if filterText := m.filterText(); filterText != "" {
+		rows = make([]Row, 0)
+		for _, row := range m.rows {
+			for _, field := range row.Fields {
+				if s, ok := field.(string); ok && strings.Contains(s, filterText) {
+					rows = append(rows, row)
```

Comment:
Suggest searching on the RenderedFields instead, to also include columns like status

```suggestion
			for _, field := range row.RenderedFields {
				if strings.Contains(field, filterText) {
					rows = append(rows, row)
```